src/packages.lisp

```diff
@@ -298,24 +352,29 @@ local src-path and sys-path."
                    (set-trans-dep d (dependency-name dep)))
                (progn
                  (format t "~%.... Checking manifest for transitive dependency: ~S" d)
-                 (let ((manifest-package (manifest-get-by-name d)))
-                   (cond ((not manifest-package)
+                 (let ((trans-dependency
+                        (if (manifest-get-by-name d)
+                            (make-trans-dep-from-manifest (manifest-get-by-name d)
+                                                          (dependency-name dep))
+                          (car (member-if
+                                (lambda (x) (string= d (dependency-name x)))
+                                *yaml-packages*)))))
```

Comment:
There is nothing important about transitive dependencies being different than a regular dependency, so you're correct there's nothing outside of the report.

I think I originally did it that way so it was clear that a _dependency of a dependency_ failed....but that info is no more helpful than normal dependencies, so this is fine.